MLS.Agent/Startup.cs

```diff
@@ -182,8 +183,9 @@ public void ConfigureServices(IServiceCollection services)
 
                 if (StartupOptions.Mode == StartupMode.Try)
                 {
+                    var uri = new Uri(app.ServerFeatures.Get<IServerAddressesFeature>().Addresses.First());
```

Comment:
Can there be more than one?

Reply:
Since we are setting up in the Program.cs to usea single url, there will be only one here - https://github.com/dotnet/try/pull/246/files#diff-adcfc4b0634e7250308be781cbe69f63R146

Reply:
https://github.com/dotnet/try/pull/246/files#diff-7e4af26b8c2fa96387cebaf1ae6d7716R14